server/src/useCases/report/productCompatibility/utils/filtersValidation.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-console */
+module.exports = class FiltersValidation {
```

Comment:
Please add information regarding pagination to class name. If we decide to use middleware then function should also provide name